src/script/script.cpp

```diff
@@ -276,6 +282,11 @@ bool CScript::HasValidOps() const
     return true;
 }
 
+#ifdef _MSC_VER
+// Re-enable MSVC warnings after C4018 suppression in CScript::HasValidOps()
+#pragma warning(pop)
+#endif
```

Comment:
NACK on adding these. It makes the code harder to parse as a human and they don't add any value.